packages/client/tests/functional/0-legacy-ports/batch-find-unique/tests.ts

```diff
@@ -70,6 +72,10 @@ testMatrix.setupTestSuite(({ providerFlavor }) => {
       prisma.user.findUnique({ where: { email: copycat.email(4) } }),
     ])
 
+    await waitFor(() => {
+      expect(executedBatchQuery).toMatchSnapshot()
+    })
```

Comment:
This should be more resilient now, as we had failure with missing an assertion and needing to update the snapshot.